RFC-004-Bitcoin.md

```diff
@@ -0,0 +1,79 @@
+# Bitcoin Definitions
+
+- RFC-Number: 004
+- Status: Draft
+- Discussion-issue: -
+- Created on: 26 Feb. 2019
+
+# Table of contents
+
+- [Description](#description)
+- [Motivation](#motivation)
+- [Content](#content)
+- [Registry extension](#registry-extension)
+
+## Description
+
+Introduces definitions to allow the execution of RFC-003 for Bitcoin asset on the [Bitcoin Core](https://github.com/bitcoin/bitcoin/) network.
+
+TODO: Decide we if do 1 RFC that introduce general Bitcoin terms + RFC003 or 2 RFCs (one for each)
+
+This includes the exact Bitcoin scripts to be used.
+
+TODO: Do we want p2sh(p2wsh) support? (more wallets support it)
+
+TODO: Do we want p2sh native support? (more wallets support it + more straightforward than p2sh(p2wsh))
+
+TODO: based on both TODO above, decide to include version and/or BIPs to be enabled.
```

Comment:
OP_CLTV BIP
Could mention what BIP should be activated (ie, segwit)
Bitcoin Core is defined by its activated BIPs
(pseudo)decision: let's see how the doc turns out